arm/migrations/versions/9cae4aa05dd7_create_settingsui_table.py

```diff
@@ -36,7 +36,7 @@ def upgrade():
                        'save_remote_images': 1,
                        'bootstrap_skin': "spacelab",
                        'language': "en",
-                       'index_refresh': 10,
+                       'index_refresh': 2000,
```

Comment:
This is gonna save a lot of frustration for users, thanks for tracking it down! Please make sure the associated tooltip info is updated to reflect the new value